styles/components/icons/mapping.less

```diff
@@ -600,6 +600,10 @@
 // GITLAB
 .icon-set(".gitlab-ci.yml", "gitlab", @orange);
 
+// GITPOD
+.icon-set(".gitpod.yml", "gitpod", @orange);
+.icon-set(".gitpod.yaml", "gitpod", @orange);
```

Comment:
AFAIK we do now support `gitpod.yaml` [[1](https://cs.github.com/gitpod-io/gitpod?q=%22gitpod.yaml%22)]. Gitpod does not parse it if it exists and therefore I think we can remove it.

```suggestion
```

Reply:
@ShashwatNautiyal Could you update the mapping and keep only `.yml` as @filiptronicek mentioned above?